scaleway/data_source_marketplace_image_beta.go

```diff
@@ -0,0 +1,50 @@
+package scaleway
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/scaleway/scaleway-sdk-go/api/marketplace/v1"
+)
+
+func dataSourceScalewayMarketplaceImageBeta() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceScalewayMarketplaceImageReadBeta,
+
+		Schema: map[string]*schema.Schema{
+			"label": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "Exact label of the desired image",
+			},
+			"instance_type": {
+				Type:        schema.TypeString,
+				Optional:    true,
```

Comment:
[`GetLocalImageIDByLabel`](https://github.com/terraform-providers/terraform-provider-scaleway/blob/712f0973ac1beafbd0104c11c206855b6f2d928b/vendor/github.com/scaleway/scaleway-sdk-go/api/marketplace/v1/marketplace_utils.go#L56:15) returns the first image that matches. To avoid flaky in unpredictable behaviors, maybe we should make it required?

Reply:
The default value of `instance_type ` is `DEV1-S` shouldn't cover unpredictable behaviors?